Enable portable PDB generation for x-plat builds

joperezr opened the pull request:
Enable portable PDB generation for x-plat builds.

cc: @weshaggard

weshaggard commented on dir.props:
```diff
@@ -175,11 +175,10 @@
     -->
-    <DebugSymbols>false</DebugSymbols>
-    <DebugType>none</DebugType>
+    <DebugSymbols>true</DebugSymbols>
```
We should be able to just delete the setting of this property now.

joperezr replied:
Ok, will do

joperezr commented:
I addressed @weshaggard comment and squashed the fix in order to trigger a new build. I'll merge this If there is no more feedback and the builds pass.

joperezr commented:
Looks like the ~100 failing tests that failed my Ubuntu and OSX builds are not related to my change, since those tests have been failing in the past few jenkins builds
